src/App.css

```diff
@@ -28,7 +28,7 @@ body {
 }
 
 .main-body {
-    width: "80%";
+    width: 80%;
     display: "flex";
```

Comment:
you could use utility class here,

Reply:
I'll look into this. Thanks for pointing out.